src/jupyterbundle/_config/config.yaml

```diff
@@ -1,4 +1,21 @@
+parameters:
+  jupyterbundle:
+    dataframe:
+      show_method: dataframe_show
+
 services:
   jupyterbundle.widgets.JupyterWidgets:
-      tags:
-        - "widgets.factory"
+    tags:
+      - "widgets.factory"
+
+  jupyterbundle.dataframe.DataFrameShowMethod:
+    tags:
+      - {name: 'jupyterbundle.dataframe.show_method', alias: dataframe_show}
+
```

Comment:
The rest of the config is unnecessary. You just need to add `pysparkbundle` as a dependency